.docker/volumes/node_2/genesis.json

```diff
@@ -14,27 +14,25 @@
       "amount": 1000000
     }
   ],
+  "pools": [
```

Comment:
isn't the pool ID auto generated based on the chain ID, and is that amount calculated as part of the total supply?